### ISSUE TITLE
Can't pan/zoom while streaming to a DynamicMap

### ISSUE THREAD

**Issue by linar-jether** [ioam/holoviews#2441]
Sending data to a stream currently causes the view to reset.

this can be disabled through the `apply_ranges` plot option (as suggested by @philippjfr) although this messes with the initial range.

Using the pan/zoom tools should pause the view updates (can be useful when we want to view a larger time window and still see updates)

**Comment by jbednar** [comment]
This might need to be a separate tool to enable and disable auto-ranging.  In most cases with streaming data, you want the auto-ranging to stop while you investigate something, but then you'll also want to re-enable auto ranging afterwards to catch up to current values.  E.g. using pan and zoom might disable the auto-range tool, but then the user could re-select it when they are done.

**Comment by linar-jether** [comment]
Yes this the same behavior currently available with the DataRange1d(follow=True), after pressing the reset tool, the range continues to follow new data

**Comment by jbednar** [comment]
Sounds perfect.

**Comment by philippjfr** [comment]
Unfortunately not entirely, we are very deliberately not using DataRange1d because it is incompatible with the normalization system we use. I would however be in favor of a plot option that allows switching to auto-ranging, but it cannot currently be enabled by default for consistency.

**Comment by jbednar** [comment]
Is this finally a third normalization option? Two never did seem like it could cover everything... :-)

**Comment by syamajala** [comment]
I would really like this as well.

**Comment by jensschroer** [comment]
I just ran into this issue. Would be great if this could be added.
